.vscode/tasks.json

```diff
@@ -0,0 +1,31 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"label": "Test (With Nox)",
+			"group": "test",
+			"type": "process",
+			"command": "python",
+			"args": [
+				"-m",
+				"nox",
+				"-s",
+				"test"
```

Comment:
We know this is technically a strictly correct way of laying out these arrays.. however, for readability, we would suggest:
```json
"args": [
    "-m", "nox",
    "-s", "test",
]
```
This makes it significantly easier to read and parse we find.

Reply:
I'm not particularrly torn on it either way personally, so that works